### ISSUE TITLE
Error message in vscode extension debug mode

### ISSUE THREAD

**Issue by ciiiii** [withfig/fig#694]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session

### Description:
When debug a vscode extension in a fig integrated vscode, some fig related error message will logged in debug console.
```
fig: Error: Command failed: fig bg:vscode-no-active-terminals
Error: unknown command "bg:vscode-no-active-terminals" for "fig"
Run 'fig --help' for usage.
```

### Environment

```bash
Fig Version: Version 1.0.52 (B328) [ABC] 
UserShell: /bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /Users/cai/.fig/bin/fig
Accessibility: true
Number of specs: 0
SSH Integration: false
Tmux Integration: false
Keybindings path: <none>
iTerm Integration: installed!
Hyper Integration: application is not present.
VSCode Integration: installed!
Docker Integration: false
Symlinked dotfiles: true
Only insert on tab: false
Installation Script: true
PseudoTerminal Path: /opt/homebrew/bin:/Users/cai/.dvm/bin/docker/20.10.6:/opt/homebrew/sbin:/opt/homebrew/bin:/opt/homebrew/opt/node@16/bin:/Users/cai/.krew/bin:/Users/cai/.cargo/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/Library/Apple/usr/bin:/Users/cai/.fig/bin:/Users/cai/gopath/bin
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: /bin/zsh (91899) - ttys006
Current working directory: /Users/cai/github/vscode-language-detect
Current window identifier: 41227/code:91846% (com.microsoft.VSCode)
Path: /opt/homebrew/bin:/Users/cai/.dvm/bin/docker/20.10.6:/opt/homebrew/sbin:/opt/homebrew/bin:/opt/homebrew/opt/node@16/bin:/Users/cai/.krew/bin:/Users/cai/.cargo/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/Library/Apple/usr/bin:/opt/homebrew/opt/python@3.10/bin:/opt/homebrew/Cellar/node@16/16.13.1/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/opt/homebrew/opt/node@16/bin:/Users/cai/.krew/bin:/Users/cai/.cargo/bin:/Users/cai/.fig/bin:/Users/cai/gopath/bin:/Users/cai/gopath/bin
Fig environment variables:
  - TERM_SESSION_ID=EB95792B-EC2E-4561-8EED-8B2D12732BEE
  - FIG_TERM_VERSION=3
  - FIG_INTEGRATION_VERSION=5
  - FIG_CHECKED_PROMPTS=1
  - FIG_TERM=1
  - FIG_ENV_VAR=1
```

**Comment by mschrage** [comment]
Hey @ciiiii, thanks for reporting this. I think this should be a simple fix.

**Comment by mschrage** [comment]
This will be fixed in the next release (v1.0.54).

**Comment by ciiiii** [comment]
@mschrage That's great, thank for your work.
